openchs-server-api/src/main/java/org/openchs/web/EncounterTypeController.java

```diff
@@ -182,12 +182,19 @@ private FormMapping createFormMapping() {
     @Transactional
     public ResponseEntity voidEncounterType(@PathVariable("id") Long id) {
         OperationalEncounterType operationalEncounterType = operationalEncounterTypeRepository.findOne(id);
+        List<FormMapping> formMappings = formMappingRepository.findAllByIsVoidedFalse();      
```

Comment:
You should ideally only be retrieving form mappings that need to be voided, not everything.